fineract-provider/dependencies.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
             'org.apache.poi:poi-ooxml-schemas',
             'org.apache.tika:tika-core',
 
+            'mysql:mysql-connector-java:8.0.23',
             'org.drizzle.jdbc:drizzle-jdbc',
```

Comment:
I think we should first get FINERACT-1177 (#1646) merged; after that above problem will be a lot easier to solve, because we'll be able to just drop external drivers like Mysql Connector or MariaDB in a folder next to the fineract-provider Spring Boot uber-jar. FINERACT-1177 doesn't require any editing of fineract-provider/dependencies.gradle. I think this is important, because as soon as we have any non-compliant JARs referenced in any \*.gradle file we are not compliant with the Apache License anymore (my 2cents, I'm not a lawyer).

Reply:
I agree.